node/src/cli.rs

```diff
@@ -62,8 +62,8 @@ pub struct AvnRunCmd {
     pub avn_port: Option<String>,
 
     /// URL for connecting with an ethereum node
-    #[arg(long = "ethereum-node-url", value_name = "ETH URL")]
-    pub eth_node_url: Option<String>,
+    #[arg(long = "ethereum-node-url", value_name = "ETH URL", num_args = 0..=5)]
```

Comment:
Why do we limit it to 5 here?

Reply:
what limit you want to have?

Reply:
I would not put a limit here

Reply:
I mean you can have up to 16 instances, so how about I put 32 as a limit?